src/it/scala/org/broadinstitute/monster/clinvar/ClinVarPipelineIntegrationSpec.scala

```diff
@@ -1,107 +1,60 @@
 package org.broadinstitute.monster.clinvar
 
 import better.files.File
-import com.spotify.scio.testing.PipelineSpec
-import io.circe.Json
-import org.apache.beam.sdk.options.PipelineOptionsFactory
-import org.scalatest.BeforeAndAfterAll
-import org.scalatest.matchers.should.Matchers
-
-class ClinVarPipelineIntegrationSpec
-    extends PipelineSpec
-    with Matchers
-    with BeforeAndAfterAll {
-  behavior of "ClinVarPipeline"
+import org.broadinstitute.monster.common.PipelineBuilderSpec
```

Comment:
I'll be able to import PipelineBuilderSpec to the encode-ingest pipeline as well, right? Instead of using all of the above boilerplate you just removed

Reply:
Yes. You'll need to bump the plugins version to 0.9